src/Monocle/Backend/Queries.hs

```diff
@@ -714,14 +717,22 @@ getAndSortToPeerStrength limit authors_peers =
      in
       toPSR <$> peers_authors
 
+-- This function is used by the API peersStrength metrics (QUERY_TOP_AUTHORS_PEERS)
```

Comment:
You might want to prefix document comment with `-- |` so that they appears in haddock:

```suggestion
-- | This function is used by the API peersStrength metrics (QUERY_TOP_AUTHORS_PEERS)
```